APP-MANAGER

```diff
@@ -151,12 +176,69 @@ else
 	_appman
 fi
 
+# Set header authorization if GitHub API key file exists
+[ -f "$AMPATH/ghapikey.txt" ] && HeaderAuthWithGITPAT=" --header \"Authorization: token $(<"$AMPATH/ghapikey.txt")\" "
+
+# DEVELOPER MODE
+if test -f "$AMPATH"/betatester; then
+	AMREPO="https://raw.githubusercontent.com/ivan-hc/AM/dev"
+	AMBRANCH=$(basename "$AMREPO")
+fi
+function _betatester_message_on() {
+	if test -f "$AMPATH"/betatester; then
+		echo '--------------------------------------------------------------------------'
+		echo "\"$AMCLI\" $AMVERSION: developer mode" | tr a-z A-Z
+		echo '--------------------------------------------------------------------------'
+	fi
+}
+
+function _use_betatester_on() {
+	touch "$AMPATH"/betatester
+}
+
+function _use_betatester_off() {
+	rm -f "$AMPATH"/betatester
+}
+
+# DOWNLOAD PORTABLE DEPENDENCES
+function _portable_dependences_setup() {
+
+	function _jq_installer() {
+		if ! command -v jq 1>/dev/null; then
+
+			_system_package_manager_check
+
+			if [[ -z "$osPKGman" ]];then
+				echo -e " ${RED}💀 ERROR!\033[0m Missing essential command ${Green}jq\033[0m\n"
+				read -p " Do you wish to install it using $AMCLI (Y,n)? " yn
+				case $yn in
+				'N'|'n')
+					echo -e "\n The \"jq\" command is essential for parsing JSON files on sites"
+					echo " for example api.github.com, to list the URLs of apps to download."
+					echo -e " You need to install it at least via your package manager to use \"$AMCLI\".\n\n Missing essential command ${RED}jq\033[0m, aborted!"; exit
+					;;
+				'Y'|'y'|*)
+					rm -f "$AMPATH"/.cache/*
+					echo ""
+					source "$AMPATH"/modules/install.am -i jq 2>&1 | grep "INSTALLED"
+					echo ""
+					;;
+				esac
+			else
+				echo -e " ${RED}💀 ERROR!\033[0m Missing essential command ${Green}jq\033[0m\n"
+				echo " Install it from your system package manager \"$osPKGman\" and retry!"; exit
```

Comment:
echo " Install it from your system package manager \"$osPKGman\"? and retry!"
exit